src/request/create/CreateHigh.js

```diff
@@ -0,0 +1,114 @@
+        this.$validateWords = (document.getElementById(CreateHigh.Pages.VALIDATE_WORDS));
+
+        /** @type {HTMLDivElement} */
+        this.$setPassphrase = (document.getElementById(CreateHigh.Pages.SET_PASSPHRASE));
+
+        /** @type {HTMLFormElement} */
+        const $passphraseSetter = (this.$setPassphrase.querySelector('.passphrase-setter'));
+
+        /** @type {HTMLFormElement} */
+        const $passphraseGetter = (this.$setPassphrase.querySelector('.passphrase-getter'));
+
+        /** @type {HTMLDivElement} */
+        const $passphraseConfirm = (this.$setPassphrase.querySelector('.confirm'));
+
+        // create components
+        this._chooseIdenticon = new ChooseIdenticon(this.$chooseIdenticon);
+        this._recoveryWords = new RecoveryWords(this.$recoveryWords);
+        this._validateWords = new ValidateWords(this.$validateWords);
+        this._passphraseSetter = new PassphraseInput(true, $passphraseSetter);
+        this._passphraseGetter = new PassphraseInput(false, $passphraseGetter);
+
+        // wire up logic
+        this._chooseIdenticon.on(
+            ChooseIdenticon.Events.CHOOSE_IDENTICON,
+            /** @param {Nimiq.KeyPair} keyPair */
+            keyPair => {
+                this._selectedKeyPair = keyPair;
+                const keyAsUInt8 = keyPair.privateKey.serialize();
+                this._recoveryWords.privateKey = keyAsUInt8;
+                this._validateWords.privateKey = keyAsUInt8;
+                this._validateWords.reset();
+                window.location.hash = CreateHigh.Pages.PRIVACY_AGENT;
+            },
+        );
+
+        this._recoveryWords.on(RecoveryWords.Events.CONTINUE, () => {
+            window.location.hash = CreateHigh.Pages.VALIDATE_WORDS;
+        });
+
+        this._validateWords.on(ValidateWords.Events.BACK, () => {
+            window.location.hash = CreateHigh.Pages.RECOVERY_WORDS;
+        });
+
+        this._validateWords.on(ValidateWords.Events.VALIDATED, () => {
+            window.location.hash = CreateHigh.Pages.SET_PASSPHRASE;
+        });
+
+        this._passphraseSetter.on(
+            PassphraseInput.Events.PASSPHRASE_ENTERED,
+            /** @param {string} passphrase */ passphrase => {
+                this._passphrase = passphrase;
+                this._passphraseSetter.reset();
+                $passphraseConfirm.classList.remove('display-none');
+                $passphraseSetter.classList.add('display-none');
+            },
+        );
+
+        this._passphraseGetter.on(
+            PassphraseInput.Events.PASSPHRASE_ENTERED,
+            /** @param {string} passphrase */ async passphrase => {
+                if (this._passphrase !== passphrase) {
+                    this._passphraseGetter.onPassphraseIncorrect();
+                    this._passphraseGetter.reset();
+                    $passphraseConfirm.classList.add('display-none');
+                    $passphraseSetter.classList.remove('display-none');
```

Comment:
I don't think it's good ux to go back to the password setter.
The user should get some message.
Also, `onPassphraseIncorrect` triggers a shake animation that you're not waiting for